.github/workflows/package_updates.yml

```diff
@@ -25,3 +25,6 @@ jobs:
         git config --global user.name "Termux Github Actions"
         git config --global user.email "contact@termux.com"
         bash ./scripts/bin/update-packages
+        
+        echo -n "\n<=====Running update check for neovim-nightly=====>"
+        bash ./scripts/bin/update-neovim-nightly
```

Comment:
Instead of doing something like this, why not support having a `termux_package_latest_version` function in `build.sh` which can be used to fetch the latest version. If this is implemented, some other packages can also be auto-updated in a similar way

Reply:
Yes, I would like to support. But could you explain more about how it would be used? I need some more context to work on it.

Reply:
I'm working on it.

Reply:
@thunder-coding have a look.